sdk/mobile-center-distribute/src/main/res/values/mobile_center_distribute.xml

```diff
@@ -15,4 +15,7 @@
     <string name="mobile_center_distribute_install_ready_message">%1$s %2$s (%3$d) has been downloaded and is ready to be installed.</string>
     <string name="mobile_center_distribute_install">Install</string>
     <string name="mobile_center_distribute_notification_category">App updates</string>
-</resources>
\ No newline at end of file
+    <string name="mobile_center_distribute_update_failed_dialog_title">In-app updates disabled</string>
```

Comment:
Were those texts and the UI reviewed by a PM? Were designers in loop?

Reply:
@guperrot everyone is aware of the changes we are making to the SDK's. The experience should be close to exactly the same on both platforms.

Reply:
yes, these strings are the same ones used in the iOS PR (https://github.com/Microsoft/mobile-center-sdk-ios/pull/773/files) which has been looked at by designers already

Reply:
Thanks